index.js

```diff
@@ -215,10 +216,16 @@ const Swipeout = React.createClass({
     if (this.state.swiping) {
       if (openRight && contentPos < 0 && posX < 0) {
         // open swipeout right
+        if(this.props.onSwipe){
```

Comment:
maybe instead of deliberate if you can just single line it out like this;

`this.props.onSwipe && this.props.onSwipe(_args_)`